lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextRuntime.cs

```diff
@@ -64,6 +69,7 @@ internal sealed class ContextRuntime : IDisposable
                 Optional<ContextRuntime> parentContext)
         {
             _serviceInjector = serviceInjector;
+            _injectedServices = serviceInjector.GetNamedInstance<ServicesSet, ISet<object>>();
```

Comment:
I understand, but how would you pass in the `childContextConfiguration` as a parameter if `ContextRuntime` is fully injectable? We will need to fork an injector with `childContextConfiguration`, so it has to be one of the constructor arguments. I would like to avoid creating a new configuration just to inject `ContextRuntime`. On the Java side, `ContextRuntime` is also not injected.